### ISSUE TITLE
temboard agent will not register fails with: CRITI: Missing discover data. Please upgrade agent.

### ISSUE THREAD

**Issue by bsislow** [dalibo/temboard#1296]
temboard version 8.0
temboard agent version 7.11

followed these directions:
https://temboard.readthedocs.io/en/v7/agent_install/

and got to `temboard-agent-register` only to hit:
```
 INFO: Registering instance/agent to https://xxx.xxx.xxx.xxx:8888 ...
CRITI: Missing discover data. Please upgrade agent.                    <---------- issue at hand
```

there is another agent installed on this host that works _fine_ on a different port; we're running 2 postgres clusters on the same host under different ports.

the agents were registered under different/unique ports as well.

please advise.

**Comment by bersace** [comment]
Hi @bsislow , temboard-agent-register is 7.11 and is not compatible with UI 8.0 new registration. This is a bug. Thanks for the report. We may fix this for 8.2.

Can you use the new UI-side `temboard register-instance` command meanwhile ?
Or better, can you ugprade the agent to 8.1 ?

Regards,
Étienne

**Comment by bsislow** [comment]
My colleague was able to purge and redeploy the agent.
